### PR TITLE
Chore(New Charts): Add charts for vm-poweroff experiment

### PR DESCRIPTION
**Pull request by iassurewipro**
Signed-off-by: Delphine Joyneer <golkonda.joyneer@wipro.com>

- Adding charts for vm-delete experiment which includes experiment CR and csv files.

### CODE REVIEW

**Review by uditgaurav — DISMISSED** [review]

File: charts/vmware/vm-delete/experiment.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          key: VCENTERPASS
+
+    - name: ANSIBLE_STDOUT_CALLBACK
+      value: 'default'
+
+    - name: TOTAL_CHAOS_DURATION
+      value: '' 
+
+    - name: LIB
+      value: ''
```
Comment (uditgaurav) [inline charts/vmware/vm-delete/experiment.yaml]: You can provide litmus here

File: charts/vmware/vm-poweroff/experiment.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    stops a vm for a certain chaos duration
+kind: ChaosExperiment
+metadata:
+  name: vm-poweroff
```
Comment (ispeakc0de) [inline charts/vmware/vm-poweroff/experiment.yaml:7]:
```suggestion
  name: vm-poweroff
  labels:
    name: vm-poweroff
    app.kubernetes.io/part-of: litmus
    app.kubernetes.io/component: chaosexperiment
    app.kubernetes.io/version: latest
```

File: charts/vmware/vm-poweroff/experiment.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+spec:
+  definition:
+    scope: Namespaced
+    permissions:
+      - apiGroups:
+          - ""
+          - "batch"
+          - "litmuschaos.io"
+        resources:
+          - "jobs"
```
Comment (ispeakc0de) [inline charts/vmware/vm-poweroff/experiment.yaml:22]:
```suggestion
          - "jobs"
          - "events"
          - "pods/log"
          - "pods/exec"
          - "secrets"
```

File: charts/vmware/vm-poweroff/vm-poweroff.chartserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+kind: ChartServiceVersion
+metadata:
+  createdAt: 2021-04-29T10:28:08Z
+  name: vm-poweroff
+  annotations:
+    categories: vmware
+    vendor: Wipro
+    support: https://app.slack.com/client/T09NY5SBT/CNXNB0ZTN
+spec:
+  displayName: vm-delete
```
Comment (ispeakc0de) [inline charts/vmware/vm-poweroff/vm-poweroff.chartserviceversion.yaml]:
```suggestion
  displayName: vm-poweroff
```

File: charts/vmware/vmware.chartserviceversion.yaml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  version: 0.1.0
+  annotations:
+    categories: vmware
+    chartDescription: Injects chaos on vmware components
+spec:
+  displayName: VMware Chaos
+  categoryDescription: >
+    VMware contains chaos to disrupt the state of vmware resources
+  experiments:
+    - vm-delete
```
Comment (ispeakc0de) [inline charts/vmware/vmware.chartserviceversion.yaml]:
```suggestion
    - vm-poweroff
```

File: charts/vmware/vmware.chartserviceversion.yaml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  maintainers:
+    - name: Delphine Joyneer
+      email: golkonda.joyneer@wipro.com
+  provider:
+    name: Wipro
+  links:
+    - name: Kubernetes Website
+      url: https://kubernetes.io
+    - name: Source Code
+      url: https://github.com/litmuschaos/litmus-go/tree/master/experiments/vm-poweroff
```
Comment (ispeakc0de) [inline charts/vmware/vmware.chartserviceversion.yaml]:
```suggestion
      url: https://github.com/litmuschaos/litmus-go/tree/master/experiments/vmware
```

File: charts/vmware/vmware.package.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+packageName: vm-poweroff
+experiments:
+  - name: vm-poweroff
+    CSV: vm-poweroff.chartserviceversion.yaml
+    desc: "vm-poweroff"
+  - name: vm-poweroff
+    CSV: vm-poweroff.chartserviceversion.yaml
+    desc: "vm-poweroff"
```
Comment (ispeakc0de) [inline charts/vmware/vmware.package.yaml]:
```suggestion
packageName: vmware
experiments:
  - name: vm-poweroff
    CSV: vm-poweroff.chartserviceversion.yaml
    desc: "vm-poweroff"
```

File: charts/vmware/vm-poweroff/engine.yaml [hunk 8]
```diff
@@ -0,0 +1,47 @@
+apiVersion: litmuschaos.io/v1alpha1
```
[the remark this answers is not in the document]
Comment (iassurewipro) [reply charts/vmware/vm-poweroff/engine.yaml:1]: @ispeakc0de Changed it to vm-poweroff.png

File: charts/vmware/vmware.package.yaml [hunk 9]
```diff
@@ -0,0 +1,5 @@
+packageName: vm-poweroff
```
Comment (ispeakc0de) [inline charts/vmware/vmware.package.yaml]:
```suggestion
packageName: vmware
```

**Review by ispeakc0de — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**